### ISSUE TITLE
Special Characters are being removed

### ISSUE THREAD

**Issue by vbtrek** [KoljaWindeler/ics#18]
Hi, I have noticed that special characters are being stripped from the Description, so my calendar appointment is "Betws-y-Coed (Caravan Holiday)" this comes through as a description of "BetwsyCoed Caravan Holiday". Is there anything that can be done to stop this?

Thanks

**Comment by andreasbrett** [comment]
Same with umlauts (ä, ö, ü etc) that are converted to "something". Example: "wöchentliche Leerung" => "wuchentliche Leerung"

**Comment by vbtrek** [comment]
It looks like the issue is caused by the [fix_text function in sensor.py](https://github.com/KoljaWindeler/ics/blob/c90d3bc50d79371bfe29f64705be25b482e81055/custom_components/isc/sensor.py#L116). If I comment out the s = ''join line then all is good for me. I'm not sure what the reason behind this method is. @andreasbrett there is also specific code in this method to replace umlauts.

**Comment by KoljaWindeler** [comment]
Hi, I had lots of issues with special chars and never figured it out. 
If you find a solution please send a pull request

**Comment by KoljaWindeler** [comment]
Ach ich glaube ich erinnere mich. Utf8 kodierte Umlaute wurden von HA fehlerhaft angezeigt.. glaube ich :D

**Comment by GLehnhoff** [comment]
Yes, not only German "Umlaute" will not be shown correctly, I also have the challenge that I would like to have the name of the day in German language. How is this possible?

**Comment by KoljaWindeler** [comment]
https://community.home-assistant.io/t/ics-custom-component-to-read-online-stored-calendars/183021/15?u=jkw

**Comment by jose1711** [comment]
> Hi, I had lots of issues with special chars and never figured it out. If you find a solution please send a pull request

@KoljaWindeler i think the issue is expecting iso8859-1 encoding in the ics file:

https://github.com/KoljaWindeler/ics/blob/b8594f062a95930628ade226b5b195fb5bbc0538/custom_components/isc/const.py#L248

Afterwards you may also drop the fix summary function.
